src/utils/get-series-podcasts.js

```diff
@@ -0,0 +1,18 @@
+import { fuzzySlugMatch } from './fuzzy-slug-match';
+
+// given a page slug, find all occurrances in "allSeries"
+export const getSeriesPodcasts = (allSeries, targetSlug) => {
```

Comment:
This is great, curious if we can optimize (later) by creating one mapping of article slug to its series to avoid continuing to iterate and find.

Reply:
Also curious if we want to test this one?